mantidproject/mantid
# Remove qt3 from Table widget in MantidPlot

Fixes #15314

Table is refactored to use QTableWidget (Qt4).

**To test**
- Run all the commands in all the menus related to tables. (Excluding cell value calculation which will be dealt with in #15316.
- Check that sample log plotting works.
- Check that imported table workspaces still work.
- Anything else that can test that behaviour of tables didn't change.
